.github/workflows/ci.yml

```
@@ -0,0 +1,123 @@
+  test:
+    name: Run Tests
+    runs-on: macos-latest
+    
+    strategy:
+      matrix:
+        destination: 
+          - 'platform=iOS Simulator,name=iPhone 16,OS=latest'
+          - 'platform=macOS'
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)